exonum-java-binding/core/src/main/java/com/exonum/binding/core/transport/VertxServer.java

```diff
@@ -85,6 +88,22 @@ public void mountSubRouter(String mountPoint, Router subRouter) {
     }
   }
 
+  @Override
+  public void removeSubRouter(String mountPoint) {
+    synchronized (lock) {
+      checkNotStopped();
+      rootRouter.getRoutes()
+          .stream()
+          .filter(r -> r.getPath().equals(mountPoint))
```

Comment:
looks like it should work well because service paths are unique within the network by design